README.md

```diff
@@ -25,34 +10,30 @@ values will be reverted to their default value (which is usually "null").
 
 ## Prerequisites
 
-This action requires:
-
--   Google Cloud credentials that are authorized to deploy a Cloud Function. See
-    the [Authorization](#Authorization) section below for more information.
-
--   [Enable the Cloud Functions API](http://console.cloud.google.com/apis/library/cloudfunctions.googleapis.com)
-
+-   This action requires Google Cloud credentials that are authorized to access
+    the secrets being requested. See [Authorization](#authorization) for more
+    information.
 
 -   This action runs using Node 16. If you are using self-hosted GitHub Actions
-    runners, you must use runner version [2.285.0](https://github.com/actions/virtual-environments)
-    or newer.
+    runners, you must use runner version
+    [2.285.0](https://github.com/actions/virtual-environments) or newer.
 
 
 ## Usage
 
 ```yaml
 jobs:
   job_id:
-    runs-on: ubuntu-latest
+    runs-on: 'ubuntu-latest'
     permissions:
       contents: 'read'
       id-token: 'write'
 
     steps:
-    - uses: actions/checkout@v3
+    - uses: 'actions/checkout@v3'
 
-    - id: auth
-      uses: google-github-actions/auth@v0
+    - id: 'auth'
+      uses: 'google-github-actions/auth@v1'
```

Comment:
@verbanicm recommended bumping v0 refs for the action being updated too.

Reply:
I have some other minor things I want to do in a follow-up PR (maybe tomorrow) - I'll bump it in that one.